css/_footer.scss

```diff
@@ -0,0 +1,60 @@
+footer{
+    background-color: #FFB6B6;
+    width: 100vw;
+    height: 35vh;
+    position: relative;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    p{
+        font-family: "Red Rose";
```

Comment:
This should probably have a fall-back font, like `sans-serif` or `serif`, or if it's a decorative/handwriting font (or monospace), some of those. [See options here](https://www.w3schools.com/css/css_font_fallbacks.asp).